tests/integration/devfile/cmd_devfile_storage_test.go

```diff
@@ -66,6 +66,50 @@ var _ = Describe("odo devfile storage command tests", func() {
 			}
 		})
 
+		It("should create storage and attach to specified container successfully and list it correctly", func() {
```

Comment:
please also add deletion to this scenario to avoid https://github.com/openshift/odo/pull/4595#issuecomment-820954677

Reply:
Added, will push shortly